Sources/Components/BottomSheet/Controller/BottomSheet.swift

```diff
@@ -0,0 +1,86 @@
+//
+//  Copyright © 2018 FINN.no. All rights reserved.
+//
+
+import UIKit
+
+public class BottomSheet: UIViewController {
+
+    // MARK: - Public properties
+
+    public var rootViewController: UIViewController
```

Comment:
I think it should be private or at least let because if you set new root view controller from outside it's gonna break things.

Reply:
We can make it private for now and then change things as we need